### PR TITLE
Fixes #33580 - Suggest new --puppet-server-foreman-url with hostname …

### PR DESCRIPTION
**Pull request by wbclark**
…change

### CODE REVIEW

**Comment by wbclark** [comment]
I think I see why this is breaking:

https://github.com/theforeman/obal/blob/ead56f57e7e3da5763482e3ede3143b458974190/obal/data/modules/rpmspec_changelog_check.py#L45-L47

It expects a specifically formatted changelog entry for the release bump

File: packages/katello/katello/katello.spec [hunk 1]
```diff
@@ -132,6 +132,9 @@ Provides a federation of katello services
 # the files section is empty, but without it no RPM will be generated
 
 %changelog
+* Tue Sep 28 2021 William Bradford Clark <wclark@redhat.com> 4.3.0-0.2master
```
Comment (ekohl) [inline packages/katello/katello/katello.spec:135]:
```suggestion
* Tue Sep 28 2021 William Bradford Clark <wclark@redhat.com> 4.3.0-0.2.master
```

**Review by ekohl — CHANGES_REQUESTED** [review]
:+1: for the change in general

**Comment by wbclark** [comment]
> +1 for the change in general

Fixed the typo... Thank you!

**Review by ekohl — APPROVED** [review]

**Pull request merged**